.github/workflows/daily.yml

```
@@ -0,0 +1,226 @@
+          slack-message: "Superface Daily: Install CLI on Linux Failed"
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+
+  install_cli_windows:
+    name: Install CLI on Windows
+    runs-on: windows-latest
+    steps:
+      - name: Use Node.js latest
+        uses: actions/setup-node@v3
+        with:
+          node-version: latest
+      - name: Install Superface CLI
+        run: npm install --global @superfaceai/cli@latest
+      - run: superface --version
+      - run: superface whoami
+      - name: Report to Slack
+        if: failure()
+        uses: slackapi/slack-github-action@v1
+        with:
+          channel-id: ${{ secrets.SLACK_CHANNEL }}
+          slack-message: "Superface Daily: Install CLI on Windows Failed"
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+    
+  create_integration:
+    name: Create integration on MacOS
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Homebrew
+        run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+      - name: Install Superface CLI
+        run: brew install superfaceai/cli/superface
+      - run: superface --version
+      - run: superface whoami
+      - run: superface prepare https://github.com/lob/lob-openapi/blob/main/lob-api-public.yml lob
+      - name: Manually update provider
+        run: |
+          echo '{
+            "name": "lob",
+            "defaultService": "default",
+            "services": [
+              {
+                "id": "default",
+                "baseUrl": "https://api.lob.com"
+              }
+            ],
+            "securitySchemes": [
+              {
+                "id": "lobBasicAuth",
+                "type": "http",
+                "scheme": "basic"
+              }
+            ],
+            "parameters": []
+          }' > superface/lob.provider.json
+      - run: superface new lob "add a new address" address-management/new-address
+      - run: superface map lob address-management/new-address
+      - run: superface map lob address-management/new-address python
+      - run: |
+          sed -i -s "s/assetsPath.*/assetsPath: '\.'/" superface/*.mjs
+      - run: |
+          sed -i -s 's/assets_path.*/assets_path = "\."/' superface/*.py
+      - name: Upload superface folder
+        uses: actions/upload-artifact@v3
+        with:
+          name: superface-folder
+          path: | 
+            superface/*.profile
+            superface/*.map.js
+            superface/*.provider.json
+            superface/*.mjs
+            superface/*.py
+            superface/package.json
+            superface/requirements.txt
+      - name: Report to Slack
+        if: failure()
+        uses: slackapi/slack-github-action@v1
+        with:
+          channel-id: ${{ secrets.SLACK_CHANNEL }}
+          slack-message: "Superface Daily: Install CLI on Linux Failed"
```

nitpick: Copy&paste typo?

```suggestion
          slack-message: "Superface Daily: Install CLI on macOS Failed"
```

---

Thanks, I rephrased it, so it says that creting integration failed.